src/bson.rs

```diff
@@ -1076,148 +1073,6 @@ impl Binary {
     }
 }
 
-/// Enum of the possible representations to use when converting between [Uuid](https://docs.rs/uuid/0.8.2/uuid/) and [`Binary`].
```

Comment:
this was all moved to the `uuid` module. The main difference is that these methods now work with `bson::Uuid` instead of `uuid::Uuid`, and that they also can be used without a feature flag.